api/registrations/views.py

```diff
@@ -799,8 +810,17 @@ def get_default_queryset(self):
     def get_queryset(self):
         return self.get_queryset_from_request()
 
+    def perform_create(self, serializer):
+        target = serializer.validated_data['target']
+        self.check_object_permissions(self.request, target)
+
+        if not target.provider.is_reviewed:
+            raise Conflict(f'{target.provider.name } is an umoderated provider. If you believe this is an error, contact OSF Support.')
+
+        serializer.save(user=self.request.user)
 
-class RegistrationRequestList(JSONAPIBaseView, ListFilterMixin, generics.ListAPIView, RegistrationMixin, ProviderMixin):
+
+class RegistrationRequestList(JSONAPIBaseView, ListFilterMixin, generics.ListCreateAPIView, RegistrationMixin, ProviderMixin):
```

Comment:
So, what's the current story with RegistrationRequests? We can just delete this since we're not following the Preprint withdrawal pattern, correct?

Reply:
I think we decided to just the request endpoints there for now, but not add POST to them